src/components/SQForm/SQFormDropdown.js

```diff
@@ -79,7 +79,11 @@ function SQFormDropdown({
       >
         {options.map(option => {
           return (
-            <MenuItem key={option.value} value={option.value}>
+            <MenuItem
+              key={option.value}
+              disabled={option.isDisabled}
```

Comment:
We should add this `isDisabled` key to the `children` propTypes shape as optional

Reply:
This property will be used if we provide isDisbaled in the options
This is not passing as a prop in SQFormDropdown 
example: const MOCK_STATE_OPTIONS = [
  {label: 'Arizona', value: 'AZ', isDisabled: true},
  {label: 'Kansas', value: 'KS',  isDisabled: true},
];

Reply:
It's being passed as an optional part of the children to SQFormDropdown. SQFormDropdown accepts it's dropdown options as children.